cockroachcloud/quickstart.md

```diff
@@ -54,17 +70,127 @@ The **Connect to cluster** dialog shows information about how to connect to your
 
 </section>
 
-<section class="filter-content" markdown="1" data-scope="go">
+<section class="filter-content" markdown="1" data-scope="java">
 
 {% include cockroachcloud/quickstart/get-connection-string.md %}
 
+1. If you haven't already, install the [`cockroach` binary](../stable/install-cockroachdb.html) and add it to your OS's `PATH`.
+
+</section>
+
+</section>
+
+<section class="filter-content" markdown="1" data-scope="local">
+
+## Step 1. Get CockroachDB
```

Comment:
Instead of having 1 substep here, can we just make this substep 1 of "Start CockroachDB"?

Reply:
Done.